packages/ui/src/component/interactive/UIInteractive.ts

```
@@ -159,6 +162,28 @@
     }
   }
 
+  // @ts-ignore
+  override _cloneTo(target: UIInteractive, srcRoot: Entity, targetRoot: Entity): void {
+    const transitions = this._transitions;
+    for (let i = 0, n = transitions.length; i < n; i++) {
+      const srcTransition = transitions[i];
+      const dstTransition: Transition<any, any> = new (<any>transitions[i]).constructor()
```

ESLint

[error] Insert `;`

(prettier/prettier)